core/modules/user/user.install

```diff
@@ -1229,6 +1229,18 @@ function user_update_1018() {
   $config->save();
 }
 
+/**
+ * Explicitly set the "user_admin_role" attribute to "0" (disabled) if it's
+ * missing.
```

Comment:
The first line of any function docblock should be a stand-alone sentence (followed by a blank line if there is more documentation). Let's shorten this up just by removing "Explicitly".

```suggestion
 * Set the "user_admin_role" attribute to "0" (disabled) if it's missing.
```